bin/cli.js

```diff
@@ -2,8 +2,12 @@
 'use strict';
 const { gatherTelemetryForUrl, analyzeEmberObject } = require('ember-codemods-telemetry-helpers');
 
+const argv = require('yargs').argv;
```

Comment:
Should probably configure more parsing so we can give nicer error messages when the correct patterns aren't setup, also can emit `--version` and `--help` output "for free".

Reply:
is that w/in the scope of this PR though?

Reply:
Yes @tomwayson, the yargs parsing was brought as part of this PR, I will make the corrections accordingly

Reply:
Ya, I'm fine if we punt the `--help` / `--version` bits to a different PR, but this PR should definitely define the specific arguments we **do** expect (e.g. `--telemetry`, and the positional values for the globs).